src/app/designationEditor/designationEditor.component.js

```diff
@@ -93,17 +95,28 @@ class DesignationEditorController {
       // get designation photos
       this.designationEditorService.getPhotos(this.designationNumber)
     ]).then(responses => {
-      this.contentLoaded = true
```

Comment:
I removed this because `this.updateCarousel` synchronously sets `this.contentLoaded` to `false`. It made tests confusing because in the test you would expect `this.contentLoaded === true` unless you check the body of `this.updateCarousel`. But maybe it would be better to remove the `this.updateCarousel` here and rely on the fact that `this.contentLoaded` is changed in this `.then()` handler to refresh the carousels. @wrandall22 Do you have thoughts on which approach is clearer?

Reply:
I think calling the function is clearer to me